docs/panel/advanced/docker.mdx

```diff
@@ -0,0 +1,203 @@
+
+## Basics
+The easiest deployment method is using the standard compose.yml file.
+
+This configuration includes an integrated web server that will automatically obtain SSL certificates if you are serving over HTTPS. For the database, it assumes you want to use SQLite (or you have an external database server to configure using the  installer.) It also assumes you intend to use the Filesystem driver for cache, filesystem or database driver for session, and database driver for queue (or you have an external Redis server to configure using the installer.) If you want to use other options built into Docker, see [Advanced Options](#advanced-options).
+
+### Create compose.yml
+```sh
+mkdir -p pelican/panel
+cd pelican/panel
+nano compose.yml
+```
+
+Now, paste the following into the compose.yml file:
+
+```yml
+services:
+  panel:
+    image: ghcr.io/pelican-dev/panel:latest
+    restart: always
+    networks:
+      - default
+    ports:
+      - "80:80"
+      - "443:443"
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+    volumes:
+      - pelican-data:/pelican-data
+      - pelican-logs:/var/www/html/storage/logs
+    environment:
+      XDG_DATA_HOME: /pelican-data
+      APP_URL: "http://localhost"
+      ADMIN_EMAIL: "USEYOUROWNEMAILHERE@example.com"
+
+volumes:
+  pelican-data:
+  pelican-logs:
+
+networks:
+  default:
+    ipam:
+      config:
+        - subnet: 172.20.0.0/16
+```
+
+### Set Required Environment Variables
+1. Set `APP_URL` to the base URL your server will be reachable on, including the protocol (https:// or http://) and port. 
+    - Note that Caddy, the integrated webserver, will serve a 308 redirect to any requests on port 80 if the `APP_URL` begins with https://. If your final site will be reachable over HTTPS but SSL will be handled and terminated by an upstream server, such as a reverse proxy, you will need to use a [custom caddyfile](#custom-caddyfile).
+2. Set the `ADMIN_EMAIL` to your email address. Caddy will use this email address to generate a LetsEncrypt SSL certificate if you are serving using HTTPS.
+
+Now, close and save changes to `compose.yml` by pressing `ctrl`+`x` then `enter`. 
+
+### Start The Panel
+From the directory in which the Caddyfile is located, run:
+
+```sh
+docker compose up -d
+```
+
+### Back Up Your Encryption Key
+The first time the container starts, it will generate an `APP_KEY` which is used as an encryption key. This will be saved automatically, but you should save a copy in a secure place in case you need it later.
+
+Run:
+
+```sh
+docker compose logs panel | grep 'Generated app key:'
+```
+
+### Install
+Open the installer in your browser at `APP_URL/installer` to finish setting up the panel.
+
+:::note
+The first time the container starts after installing or updating, it will apply database migrations. This process may take several minutes, during which the panel will not be accessible.
+:::
```

Comment:
This takes seconds on my Mac, which is much slower than on Linux.

```suggestion
```

Reply:
On my server, running in Docker using MySQL with the workaround noted in https://github.com/pelican-dev/panel/issues/986, it takes a couple of minutes to apply all the migrations. I imagine it's faster on SQLite and if the schema is able to apply?